trashtalk/apps/cleanups/constants.py

```diff
@@ -281,21 +281,14 @@ class LocationCategory:
     landnmark = 'landmark'
 
 
-class HtmlConstants:
+class ValidatorPatterns:
     # TODO: Issue #114 - Do we still need? If so, refactor as regexes for validators.
-    def __init__(self):
-        self.password_pattern = "(?=.*\d)(?=.*[a-z])(?=.*[A-Z]).{6,}"
-        self.password_title = "Must contain at least one number and one uppercase and lowercase letter, and at least 6 or more characters"
-
-        self.username_pattern = "[a-zA-Z0-9]{2,}"
-        self.username_title = "2-20 alpha-numeric characters"
-
-        self.date_pattern = "(?:19|20)[0-9]{2}-(?:(?:0[1-9]|1[0-2])-(?:0[1-9]|1[0-9]|2[0-9])|(?:(?!02)(?:0[1-9]|1[0-2])-(?:30))|(?:(?:0[13578]|1[02])-31))"
-        self.date_placeholder = "yyyy-mm-dd"
-
-        self.time_pattern = "([0-9]|0[0-9]|1[0-2])(:[0-5][0-9])"
-        self.time_placeholder = "hh:mm"
-
-        self.default_image_path = "/static/images/defaults/"
-        self.tool_image_path = "/static/images/tools/"
-        self.text_pattern = "[a-zA-Z\s]{1,}"
+    password_pattern = "(?=.*\d)(?=.*[a-z])(?=.*[A-Z]).{6,}"
+    password_title = "Must contain at least one number and one uppercase and lowercase letter, and at least 6 or more characters"
+    username_pattern = "[a-zA-Z0-9]{2,}"
+    username_title = "2-20 alpha-numeric characters"
+    date_placeholder = "yyyy-mm-dd"
+    time_placeholder = "hh:mm"
+    default_image_path = "/static/images/defaults/"
```

Comment:
We don't require this at current. Using the `static` method in templates already covers this for us.